core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala

```diff
@@ -361,6 +361,16 @@ case class DefaultRuntimeLabels(runtimeName: RuntimeName,
     ).filterNot(_._2 == null)
 }
 
+object defaultLabelKeys {
```

Comment:
if this is used for unit tests, I'd move it to https://github.com/DataBiosphere/leonardo/blob/develop/http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/CommonTestData.scala

Reply:
I think this is intended to be a companion object, so should be `object DefaultRuntimeLabels {..}`

The intent is to return the default label _keys_. Could make that more explicit by:
```
object DefaultRuntimeLabels {
  // Creating a dummy instance to obtain the default label keys
  val defaultLabelKeys = DefaultRuntimeLabels(RuntimeName(""),
                                           GoogleProject(""),
                                           WorkbenchEmail(""),
                                           WorkbenchEmail(""),
                                           None,
                                           None,
                                           None).toMap.keySet
}
```

Unless anyone has a better way..